builtin/blame.c

```diff
@@ -53,7 +53,7 @@ static const char *annotate_opt_usage[] = {
 };
 
 static int longest_file;
-static int longest_author;
+static size_t longest_author;
```

Comment:
May I suggest to do this as a pre-emptive commit, i.e. reorder it to come _before_ the signature change, and to use the `utf8:` prefix in the commit message?

Reply:
> May I suggest to do this as a pre-emptive commit, i.e. reorder it to come before the signature change

Sorry, but I don't understand the rationale behind this. What difference will this make?

Reply:
The difference is that it will provide a better structure for the benefit of any reviewer.